code/jasmin/mlkem_ref/indcpa.jinc

```diff
@@ -28,6 +28,7 @@ fn __indcpa_keypair(reg u64 pkp, reg u64 skp, reg ptr u8[MLKEM_SYMBYTES] randomn
     inbuf[u64 i] = t64;
   }
 
+  #[declassify]
```

Comment:
I think this declassifies too much? `noiseseed` is the only secret input to secret key generation.
The public seed is the first 32(?) bytes of buf. Perhaps we can declassify on copy? (Line 37.)

Reply:
Well spotted. Fixed.